content/tutorials/node/hello-world.md

```diff
@@ -11,6 +11,8 @@ In this tutorial, you will learn how to create a topic, build a producer/consume
 Examples in this section require an access to an existing Fluvio cluster and Node.js **v12.11.0** or above installed on your machine.
 <br />
 
+See [Getting Started](/docs/getting-started/) guide for installing a Fluvio cluster.
```

Comment:
I've been saying "getting access to" a Fluvio cluster, because they could also use Cloud. Not sure if it's significant

Reply:
Commit a96b82a updates text to 
```
See [Getting Started](/docs/getting-started/) guide to get access to a Fluvio cluster.
```